files/auto/autopause-daemon.sh

```diff
@@ -71,6 +76,12 @@ do
     if java_clients_connected ; then
       logAutopause "Client connected - waiting for disconnect"
       STATE=E
+    else
+      TIME_THRESH=$(($(current_uptime)+$AUTOPAUSE_TIMEOUT_KN))
+      if -e /data/.skip-pause ; then
+        logAutopause "`/data/.skip-pause` file is present - skipping pausing"
+        STATE=E
+      fi
```

Comment:
I'm thinking this should just be a simple `elif` on the file existence and if present, leave the state as-is and not reset the overall `TIME_THRESH`. BTW a state of "E" means "established connections".

Reply:
Alright, I replaced this with an `elif` as well and removed the timer reset.